var/spack/repos/builtin/packages/py-datalad/package.py

```diff
@@ -0,0 +1,107 @@
+    of data from online portals and exposing it in readily usable form as
+    Git(-annex) repositories, so-called datasets. The actual data storage and
+    permission management, however, remains with the original data providers.
+    """
+
+    homepage = "http://datalad.org/"
+    pypi     = "datalad/datalad-0.14.6.tar.gz"
+
+    version('0.14.6', sha256='149b25a00da133a81be3cbdc041a1985418f0918fa5961ba979e23b5b3c08c63')
+
+    variant('downloaders-extra', default=False)
+    variant('misc', default=False)
+    variant('tests', default=False)
+    variant('metadata-extra', default=False)
+    variant('duecredit', default=False)
+    variant('full', default=False)
+
+    depends_on('python@3.6:', type=('build', 'run'))
+    depends_on('py-setuptools@40.8.0:', type='build')
+    depends_on('git', type=('build', 'run'))
+    depends_on('git-annex', type=('build', 'run'))
+
+    # core
+    depends_on('py-appdirs', type=('build', 'run'))
+    depends_on('py-chardet@3.0.4:', type=('build', 'run'))
+    depends_on('py-distro', type=('build', 'run'), when='^python@3.8:')
+    depends_on('py-iso8601', type=('build', 'run'))
+    depends_on('py-humanize', type=('build', 'run'))
+    depends_on('py-fasteners@0.14:', type=('build', 'run'))
+    depends_on('py-patool@1.7:', type=('build', 'run'))
+    depends_on('py-tqdm', type=('build', 'run'))
+    depends_on('py-wrapt', type=('build', 'run'))
+    depends_on('py-annexremote', type=('build', 'run'))
+    # additional dependency on 'colorama' if platform_system=="Windows"'
+
+    # downloaders
+    depends_on('py-boto', type=('build', 'run'))
+    depends_on('py-keyring@8.0:', type=('build', 'run'))
+    depends_on('py-keyrings-alt', type=('build', 'run'))
+    depends_on('py-msgpack', type=('build', 'run'))
+    depends_on('py-requests@1.2:', type=('build', 'run'))
+
+    # puplish
+    depends_on('py-jsmin', type=('build', 'run'))
+    depends_on('py-pygithub', type=('build', 'run'))
+
+    # metadata
+    depends_on('py-simplejson', type=('build', 'run'))
+    depends_on('py-whoosh', type=('build', 'run'))
+
+    with when('+downloaders-extra'):
+        depends_on('py-requests-ftp', type=('build', 'run'))
+
+    with when('+misc'):
+        depends_on('py-pyperclip', type=('build', 'run'))
+        depends_on('py-python-dateutil', type=('build', 'run'))
+
+    with when('+tests'):
+        depends_on('py-beautifulsoup4', type=('build', 'run'))
+        depends_on('py-httpretty@0.9.4:', type=('build', 'run'))
+        depends_on('py-nose@1.3.4:', type=('build', 'run'))
+        depends_on('py-vcrpy', type=('build', 'run'))
+
+    with when('+metadata-extra'):
+        depends_on('py-pyyaml', type=('build', 'run'))
+        depends_on('py-mutagen@1.36:', type=('build', 'run'))
+        depends_on('py-exifread', type=('build', 'run'))
+        depends_on('py-python-xmp-toolkit', type=('build', 'run'))
+        depends_on('py-pillow', type=('build', 'run'))
```

Comment:
```suggestion
        depends_on('pil', type=('build', 'run'))
```
We have a virtual dependency for this.